WordPress/gutenberg
# Scripts: Update `sass` and `sass-loader` packages

## What?

Updates the `sass` and `sass-loader` dependencies used in `@wordpress/scripts`.

## Why?

The older version of `sass-loader` is using a legacy API that is coming to end-of-life. See https://sass-lang.com/documentation/breaking-changes/legacy-js-api/. 

The `@wordpress/scripts` build process also shows multiple warnings every time it runs when sass files exist. This update removes them.

## How?

This updates the `@wordpress/scripts package.json` file as well as the `main package.json` and `main package-lock.json` files to use the most recent updated version of sass and sass-loader.

## Testing Instructions

1. To test the changes, add the following to a package.json file that is using `@wordpress/scripts`:

```json
"overrides": {
	"sass-loader": "^16.0.2",
	"sass": "^1.79.3"
},
```
2. Run: `npm install`
3. Run the build process: `npm run build`
4. See that no issues are reported
